### ISSUE TITLE
Screen Reader: Settings–Subscription: The visual list of plan details is not announced as a list

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76951]
## Action Performed
Prerequisite:
- User is logged in

1. Using JAWS + Chrome, open the URL: https://new.expensify.com/
2. Tab to focus on the "My Settings" button and press Enter to activate it
3. Once the My Settings menu is displayed, tab to focus on the "Subscription" menu item and press Enter to activate it
4. Use the arrow keys or press the L key to navigate to the visual list in the "Your Plan" section.
5. Observe whether the visual list is announced as a list by the screen reader.

## Expected Result
The plan details should be marked up using semantic list elements or appropriate ARIA roles (e.g., role="list" and role="listitem"). This ensures that screen readers announce the group as a list, followed by each list item being read sequentially.
Example:
“Control Plan, List with 8 items, Everything in the Collect plan list item, 1 of 8”

## Actual Result
The visual list of plan details (e.g., “Everything in the Collect plan,” “Custom expense rules,” “ERP integrations”, "HR integrations (Workday, Certinia), "SAML/SSO", "Custom insights and reporting", "Budgeting") is presented visually as a list but is not programmatically identified as one.
When a screen reader navigates through these items, it reads them as separate text elements rather than part of a list structure.
As a result, users who rely on assistive technologies do not receive information about the number of items or their relationship within the list.

HTML:
<div class="css-g5y9jx r-13awgt0 r-18u37iz r-1awozwy r-1wzrnnt"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="16" height="16" hovered="false" pressed="false" fill="#03D47C"><path d="M17.2 3.4c.9.7 1.1 1.9.4 2.8L8.9 18l-6.3-5.7c-.8-.7-.9-2-.1-2.8.7-.8 2-.9 2.8-.1l3 2.7 6.1-8.2c.6-1 1.9-1.2 2.8-.5" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div><div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-left: 8px;">Everything in the Collect plan</div></div>

## Area issue was found in:
Settings–Subscription

## Failed WCAG checkpoints
WCAG 1.3.1 Info and Relationships
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

## User impact:
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested resolution:
Implement the information as an unordered list.
https://www.w3.org/WAI/tutorials/page-structure/content/#lists

## Environment(s)
Windows 11 Home - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/1dbf64fb-6b30-4b13-8f79-e2b7b7c32f11

<img width="1918" height="966" alt="Image" src="https://github.com/user-attachments/assets/03037b23-979c-4662-be53-acc4d8565dab" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999295
**Issue reported by:** NA
**Slack conversation:** NA
